releasenotes/releasenotes_v6.rst

```diff
@@ -10,13 +10,31 @@ Current Firely Server release notes (v6.x)
 
 .. _vonk_releasenotes_6_1_0:
 
-Release 6.1.0, June 15th, 2025
+Release 6.1.0, XXXX, 2025
 ------------------------------
 
 Security
 ^^^^^^^^
 
 #. AccessPolicy resources can now only be accessed or modified with system-level scopes (e.g., ``system/AccessPolicy.*``). Patient-level scopes (``patient/AccessPolicy.*``) and user-level scopes (``user/AccessPolicy.*``) are not allowed and will be rejected with a 403 Forbidden response.
+#. ``TrustedProxyIPNetworks`` now has an additional setting ``AllowAnyNetworkOrigins`` to allow any network origins to be trusted. This setting is disabled by default and should only be enabled if you are sure that your network is secure.
```

Comment:
Please add that system that previously used ASPNETCORE_ENVIRONMENT should switch to the new setting.